pkg/csi/service/common/authmanager.go

```diff
@@ -384,6 +389,14 @@ func getDsToFileServiceEnabledMap(ctx context.Context, vc *vsphere.VirtualCenter
 			log.Errorf("failed to get the vsan cluster config. error: %+v", err)
 			return nil, err
 		}
+		if !(*config.Enabled) {
+			log.Debugf("cluster: %+v is a non-vSAN cluster. Skipping this cluster", cluster)
+			continue
+		} else if config.FileServiceConfig == nil {
```

Comment:
We are dereferencing config with *config in the above if condition, do we need to do this here as well?

Reply:
same comment as above.